src/components/book.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Books(props) {
+  const { title, author } = props;
+  return (
+    <>
+      <div>
+        <ul>
+          <li>
+            <span>{title}</span>
+            <span>{author}</span>
+            <button type="submit">Remove</button>
+          </li>
+        </ul>
+        <h2>Add New Book</h2>
+        <form>
+          <div>
+            <input placeholder="Book title" />
+          </div>
+          <div>
+            <select>
+              <option selected>Category</option>
+              <option value="ONE">Category1</option>
+              <option value="TWO">categories2</option>
+            </select>
+          </div>
+          <div>
+            <button type="submit">Add Book</button>
+          </div>
+        </form>
+      </div>
+
+    </>
+  );
+}
```

Comment:
- [ ] This component should split into two. The first component should simply take in `title` and `author` as props (as you have correctly done). Its responsibility should solely be the rendering of ONE book. It should be a reusable component.

- [ ] The second component should be a form component. Kindly remove the form in this component and move it into a new component whose sole responsibility would be to add books.

- [ ] Finally, create another component (name can be `BookList`) that would render this Book component and the Form component.